src/widgets/helper/ChannelView.cpp

```diff
@@ -2027,12 +2027,15 @@ void ChannelView::mouseMoveEvent(QMouseEvent *event)
 
             if (emoteElement)
             {
-                this->tooltipWidget_->setOne({
-                    showThumbnail
-                        ? emoteElement->getEmote()->images.getImage(3.0)
-                        : nullptr,
-                    element->getTooltip(),
-                });
+                auto scale = getSettings()->emoteTooltipScale.getValue();
+                TooltipEntry entry = {
+                    .image =
+                        showThumbnail
+                            ? emoteElement->getEmote()->images.getImage(scale)
+                            : nullptr,
+                    .text = element->getTooltip()};
+                entry.scaleImg(scale);
```

Comment:
I'd prefer if the `scaleImg` function is removed.

You could add a static function to `TooltipEntry` that creates a scaled version, e.g. `auto entry = TooltipEntry::scaled(image, text, scale);` instead, or set `customWidth`/`customHeight` yourself right here